test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -7,44 +7,44 @@ Describe "TabCompletion" -Tags CI {
 
     It 'Should complete Command' {
         $res = TabExpansion2 -inputScript 'Get-Com' -cursorColumn 'Get-Com'.Length
-        $res.CompletionMatches[0].CompletionText | Should be Get-Command
+        $res.CompletionMatches[0].CompletionText | Should -Be Get-Command
```

Comment:
All Shoulds comparing string must be `-BeExactly`

Reply:
I believe we shouldn't mess syntax changes and the _functional_ changes - our tests have tons gaps like this - this huge job needs to be done in other PRs.